### PR TITLE
Added Wallet

### PR DESCRIPTION
**Pull request by Maryom**

### CODE REVIEW

File: Appz/Appz/Apps/Wallet.swift [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+//  Copyright © 2018 kitz. All rights reserved.
+//
+
+public extension Applications {
+    
+    public struct Wallet: ExternalApplication {
+        
+        public typealias ActionType = Applications.Wallet.Action
+        
+        public let scheme = "shoebox:"
```
Comment (Mazyod) [inline Appz/Appz/Apps/Wallet.swift:15]: lol, interesting name

**Comment by Mazyod** [comment]
👌

File: Appz/Appz/Apps/Wallet.swift (hunk 1 above)
Comment (Maryom) [reply Appz/Appz/Apps/Wallet.swift:15]: 😊

**Comment by Maryom** [comment]
I took it from [here](https://github.com/phynet/iOS-URL-Schemes)

**Pull request merged**